### ISSUE TITLE
NameError (uninitialized constant Mailjet::Statcounters)

### ISSUE THREAD

**Issue by lcoq** [mailjet/mailjet-gem#149]
I need to get the number of sent messages for a specific campaign.

[The documentation says that I have to use the `/statcounters` API for this](https://dev.mailjet.com/guides/?ruby#key-performance-statistics), available in this gem through :

```ruby
variable = Mailjet::Statcounters.all(source_id: "$Campaign_ID",
  counter_source: "Campaign",
  counter_timing: "Message",
  counter_resolution: "Lifetime"
)
```

But this raises a `NameError (uninitialized constant Mailjet::Statcounters)`.
I did not see any reference of this constant in this gem.

How could I achieve this ?

**Comment by adamyanliev** [comment]
Hey @lcoq there's a naming issue with `/statcounters` for Ruby in the guide. Please try using `Statcounter` instead of `Statcounters` and you should be able to find the constant. 

I'll check further whether/when the constant name will be changed and post when I have more info.

Regards,
Atanas

**Comment by adamyanliev** [comment]
Hi,

We'll look into this, it seems it was never added here after the resource was developed. 

Meanwhile if you want to retrieve the number of sent messages for a campaign, you can use `Campaignstatistics` instead. The `DeliveredCount` property here indicates the number of messages that were successfully sent and did not bounce or were blocked because of a previous bounce. `ProcessedCount` will give you information on all messages that were processed for this campaign, including blocked and bounced ones.

Regards,
Atanas

**Comment by marcel12bell** [comment]
Hi @adamyanliev  Campaignstatistics is now  a legacy endpoint. We advise using /statcounters ... when will Mailjet::Statcounter/s be implemented?

**Comment by xab3r** [comment]
Fixed in v1.6.0
